css/main.css

```diff
@@ -131,11 +132,87 @@ nav .user-greet .user-greet-msg {
   font-size: 1.125em;
 }
 
+.linkedin-btn{
+  font-size: 16px;
+  border-radius: 10px;
+  padding: .3rem;
+  background-color: white;
+  cursor: pointer;
+}
+
+.logo-linkedin{
+  vertical-align: middle;
+  width: 25px;
+  height: 25px;
+}
+
+.linkedinlink{
+  text-decoration: none;
+}
+
+
+.twitter-btn{
+  font-size: 16px;
+  border-radius: 10px;
+  padding: .3rem;
+  background-color: white;
+  cursor: pointer;
+}
+
+.logo-twitter{
+  vertical-align: middle;
+  width: 25px;
+  height: 25px;
+}
+
+.twitterlink{
+  text-decoration: none;
+}
+
+.facebook-btn{
+  font-size: 16px;
+  border-radius: 10px;
+  padding: .3rem;
+  background-color: white;
+  cursor: pointer;
+}
+
+
+.logo-facebook{
+  vertical-align: middle;
+  width: 25px;
+  height: 25px;
+}
+
+.facebooklink{
+  text-decoration: none;
+}
+
+.instagram-btn{
+  font-size: 16px;
+  border-radius: 10px;
+  padding: .3rem;
+  background-color: white;
+  cursor: pointer;
+}
+
+.logo-instagram{
+  vertical-align: middle;
+  width: 25px;
+  height: 25px;
+}
+
+.instagramlink{
+  text-decoration: none;
+}
+
```

Comment:
@sumitd94 How does this CSS look? Any issues?

Reply:
Mixture of `rem` and `px` ?

Reply:
@sumitd94 No. Look closely again.

Reply:
@ankushdharkar same and repeated css for Facebook btn, Twitter btn, LinkedIn btn etc.